### PR TITLE
Fix flake8 issues so GitHub Action goes green

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by th3coop** [comment]
@cclauss you didn't have to do that, but much appreciated!  Thanks again.

**Comment by cclauss** [comment]
Just for completeness ;-)

**Comment by cclauss** [comment]
I like green elves.

**Comment by th3coop** [comment]
> I like green elves.

Best ones by far haha.

File: elf.py [hunk 1]
```diff
@@ -93,15 +92,15 @@ def run_game_loop(self):
             if self.player.walking:
                 self.player.walk()
             # Create the event loop
-            keys = key.get_pressed()  # checking pressed keys
+            # keys = key.get_pressed()  # checking pressed keys
```
Comment (th3coop) [inline elf.py]:
```suggestion
```

Comment (th3coop) [reply elf.py]: I'm fine if you just snip that.

File: gameObjects.py [hunk 2]
```diff
@@ -149,16 +151,16 @@ def calculate_location(self, velocity):
 
     # Initialize the move but don't do it twice
     def start_move(self):
-        if self.move_start_time is 0:
+        if self.move_start_time == 0:
```
Comment (th3coop) [inline gameObjects.py:154]: Do you know why this change is suggested?  I noticed when comparing to `None` it suggested the opposite.  I'm guess it has something to do with implicit type conversion?

**Review by th3coop — APPROVED** [review]
I suggested a change but I want to see if I can just do it through the web UI for the PR.

**Comment by th3coop** [comment]
Oh duh, it's YOUR repo, not mine.  Nvm. Thanks!

File: gameObjects.py (hunk 2 above)
Comment (cclauss) [reply gameObjects.py:154]: Identity is not the same thing as equality in Python so use ==/!= to compare str, bytes, and int literals. In Python >= 3.8, these instances will raise __SyntaxWarnings__ so it is best to fix them now.  https://docs.python.org/3.8/whatsnew/3.8.html#porting-to-python-3-8

% __python__
```
>>> present = "pres"
>>> present += "ent"
>>> present == "present"
True
>>> present is "present"
False
>>> 0 == 0.0
True
>>> 0 is 0.0
False
```

Comment (cclauss) [reply gameObjects.py:154]: None, True, False are singletons in Python so using __is__ is what you want.

Comment (th3coop) [reply gameObjects.py:154]: Thank you kindly.
